tests/test_zuora_all_fields.py

```diff
@@ -0,0 +1,79 @@
+from tap_tester import connections, runner, menagerie
+from base import ZuoraBaseTest
+
+
+# These are the streams which don't support Deleted field coming in the catalog
+DOES_NOT_SUPPORT_DELETED = ['AccountingPeriod', 'ContactSnapshot', 'DiscountAppliedMetrics', 'UpdaterDetail',
+'PaymentGatewayReconciliationEventLog', 'PaymentTransactionLog', 'PaymentMethodTransactionLog', 'UpdaterBatch', 
+'PaymentReconciliationJob', 'PaymentReconciliationLog', 'ProcessedUsage', 'RefundTransactionLog', 
+'BookingTransaction', 'CalloutHistory', 'SmartPreventionAudit', 'HpmCaptchaValidationResult','EmailHistory']
+
+class ZuoraAllFields(ZuoraBaseTest):
+    """Ensure running the tap with all streams and fields selected results in the replication of all fields."""
+     
+    def name(self):
+        return "tap_tester_zuora_all_fields_test"
+
+    def test_run(self):
+        """ Executing tap-tester scenarios for both types of zuora APIs AQUA and REST"""
+         # Testing for only AQuA mode to reduce the execution time
+        self.run_test("AQUA")
+
+    def run_test(self, api_type):
```

Comment:
We can simply define `api_type` inside and keep only `test_run()` method unless I am missing something.

Reply:
Initially, executing the run_test() for both the types of APIs `REST` and `AQUA` but removed it for `REST` afterwords to reduce the execution time. Wil keep it as it is to maintain the consistency throughout the integration test cases.